qa/workunits/cephtool/test.sh

```diff
@@ -1,10 +1,10 @@
-#!/bin/bash -x
+#!/usr/bin/env bash
 # -*- mode:shell-script; tab-width:8; sh-basic-offset:2; indent-tabs-mode:t -*-
 # vim: ts=8 sw=8 ft=bash smarttab
 
 source $(dirname $0)/../ceph-helpers.sh
 
-set -e
+set -ex
```

Comment:
@asomers 
This change silences the -x on the ceph-helpers.sh, not sure you want to change behaviour in such an invasive patch.